### PR TITLE
Fix issues with date formating in case of deleting a date field

### PR DESCRIPTION
**Pull request by MariusBluem**
downstream from https://github.com/owncloud/contacts/pull/527

### CODE REVIEW

**Comment by MariusBluem** [comment]
Please check this one manually @nickvergessen - there were some problems with git 😁

File: js/models/contact_model.js [hunk 1]
```diff
@@ -145,15 +145,53 @@ angular.module('contactsApp')
 				}
 			},
 
+<<<<<<< HEAD
 			getProperty: function(name) {
 				if (this.props[name]) {
 					return this.props[name][0];
+=======
```
Comment (nickvergessen) [inline js/models/contact_model.js]: nop

File: js/models/contact_model.js [hunk 2]
```diff
@@ -145,15 +145,53 @@ angular.module('contactsApp')
 				}
 			},
 
+<<<<<<< HEAD
```
Comment (nickvergessen) [inline js/models/contact_model.js]: nop

**Comment by nickvergessen** [comment]
Please try again.
Which command did you use btw?

**Comment by MariusBluem** [comment]
Cherry Picking ... Maybe it would help to get #8 and #10 in?! Commit is causing conflicts, because the `js/models/contact_model.js` is touched in those PR(s) 😁 @nickvergessen

**Comment by MariusBluem** [comment]
I am not able to solve the conflicts in a proper way 😁 @nickvergessen

**Comment by nickvergessen** [comment]
Replaced by https://github.com/nextcloud/contacts/pull/21

**Pull request closed without merging**